Lib/email/_policybase.py

```diff
@@ -307,7 +307,29 @@ def header_source_parse(self, sourcelines):
 
         """
         name, value = sourcelines[0].split(':', 1)
-        value = value.lstrip(' \t') + ''.join(sourcelines[1:])
+
+        # Fixed: https://github.com/python/cpython/issues/124452
+        #
+        # Root cause: The function '_refold_parse_tree' in '_header_value_parse.py'.
+        # If there is no WSP, it can't figure out how to wrap the text.
+        # Therefore, it places the entire value directly after '\n', and because
+        # there is a WSP after '<HeaderName>:', the WSP will be moved to the front
+        # of the value according to RFC5322, section 2.2.3.
+        #
+        # However, the WSP is not part of the value; therefore, we must
+        # remove it.
+
+        no_first_value = value.strip() == '' and len(sourcelines) > 1
+
+        # When using the compat32 policy, the value is '\n'. Therefore,
+        # use an empty string if there is no value (without WSP and CRLF)
+        # on the first line
+        value = '' if no_first_value else value.lstrip(' \t')
+
+        if no_first_value and sourcelines[1][0] in ' \t':
+            sourcelines[1] = sourcelines[1][1:]
+
+        value += ''.join(sourcelines[1:])
```

Comment:
A much simpler fix:

```
value = ''.join((value, *sourcelines[1:])).lstrip(' \t\r\n')
```

This makes it match the *intent* of the docstring, which should be rewritten as "The value is determined by stripping the leading whitespace off the remainder of the first line joined with the all subsequent lines..."

Reply:
yeahhh, it's much easier. was trying to trim the ' \t' only.